city_scrapers/spiders/atconj_Atlantic_City.py

```diff
@@ -0,0 +1,147 @@
+import json
```

Comment:
```suggestion
```

Reply:
Good catch. I was still using `json.loads` on the meeting details method. That is now refactored. Thanks!